e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -7,6 +7,45 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+function verifyFocusInAddChannelMemberModal() {
+    // # Click to access the Channel menu
+    cy.get('#channelHeaderTitle').click();
+
+    // * The dropdown menu of the channel header should be visible;
+    cy.get('#channelHeaderDropdownMenu').should('be.visible');
+
+    // # 1. Click 'Add Members'
+    cy.get('#channelAddMembers').click();
+
+    // * Assert that modal appears
+    cy.get('#addUsersToChannelModal').should('be.visible');
+
+    // * Assert that the input box is in focus
+    cy.get('#selectItems input').should('be.focused');
+
+    // # Push a character key such as "A"
+    cy.focused().type('A');
+
+    // * Check that input box has character A
+    cy.get('#selectItems input').should('have.value', 'A');
+
+    // # Click anywhere in the modal that is not on a field that can take focus
+    cy.get('#channelInviteModalLabel').click();
+
+    // * Note the focus has been removed from the search box
+    cy.get('#selectItems').should('not.be.focused');
+
+    // # Push a character key such as "A"
+    cy.get('body').type('A');
+
+    // * Focus is not moved anywhere. Neither the search box or main input box has the focus
+    cy.get('#selectItems').should('not.be.focused');
```

Comment:
You should check if the input is not focused rather than the div. Hence can you change it to:
cy.get('#selectItems input').should('not.be.focused');

Also change the same in line 36

You can also club the last step as a single step and include it here itself. Something like this:
cy.get('#selectItems input').should('not.be.focused').and('have.value', 'A');

Reply:
good catch, I've made the changes you requested. Thank you :)